<!DOCTYPE html>
<html lang='en'>
<head>
<title>0002-load-site-font-and-site-tmac-from-etc-groff.patch - rpms/groff.git - groff</title>
<meta name='generator' content='cgit v1.1'/>
<meta name='robots' content='index, nofollow'/>
<link rel='stylesheet' type='text/css' href='/cgit-data/cgit.css'/>
<link rel='shortcut icon' href='/favicon.ico'/>
<link rel='alternate' title='Atom feed' href='http://src.fedoraproject.org/cgit/rpms/groff.git/atom/0002-load-site-font-and-site-tmac-from-etc-groff.patch?h=master' type='application/atom+xml'/>
<link rel='vcs-git' href='https://src.fedoraproject.org/git/rpms/groff.git' title='rpms/groff.git Git repository'/>
</head>
<body>
<script src="https://apps.fedoraproject.org/fedmenu/js/jquery-1.11.2.min.js"></script>
<script src="https://apps.fedoraproject.org/fedmenu/js/fedmenu.js"></script>

<script>
    var base = 'https://apps.fedoraproject.org/';
    var tokens = window.location.href.split('/');
  var repo = null;
  if (tokens.length > 5 && tokens[4] == 'rpms' && tokens[5] != '') {
    repo = tokens[5].split('.')[0];
  }
  fedmenu({
      'url': base + 'js/data.js',
      'mimeType': 'application/javascript',
      'position': 'bottom-right',
      'package': repo,
  });
</script>
<div id='cgit'><table id='header'>
<tr>
<td class='logo' rowspan='2'><a href='/cgit/'><img src='/cgit-data/cgit.png' alt='cgit logo'/></a></td>
<td class='main'><a href='/cgit/'>index</a> : <a title='rpms/groff.git' href='/cgit/rpms/groff.git/'>rpms/groff.git</a></td><td class='form'><form method='get'>
<select name='h' onchange='this.form.submit();'>
<option value='f10'>f10</option>
<option value='f11'>f11</option>
<option value='f12'>f12</option>
<option value='f13'>f13</option>
<option value='f14'>f14</option>
<option value='f15'>f15</option>
<option value='f16'>f16</option>
<option value='f17'>f17</option>
<option value='f18'>f18</option>
<option value='f19'>f19</option>
<option value='f20'>f20</option>
<option value='f21'>f21</option>
<option value='f22'>f22</option>
<option value='f23'>f23</option>
<option value='f24'>f24</option>
<option value='f25'>f25</option>
<option value='f26'>f26</option>
<option value='f27'>f27</option>
<option value='f28'>f28</option>
<option value='f7'>f7</option>
<option value='f8'>f8</option>
<option value='f9'>f9</option>
<option value='master' selected='selected'>master</option>
</select> <input type='submit' value='switch'/></form></td></tr>
<tr><td class='sub'>groff</td><td class='sub right'>Jesse Keating</td></tr></table>
<table class='tabs'><tr><td>
<a href='/cgit/rpms/groff.git/'>summary</a><a href='/cgit/rpms/groff.git/refs/'>refs</a><a href='/cgit/rpms/groff.git/log/0002-load-site-font-and-site-tmac-from-etc-groff.patch'>log</a><a class='active' href='/cgit/rpms/groff.git/tree/0002-load-site-font-and-site-tmac-from-etc-groff.patch'>tree</a><a href='/cgit/rpms/groff.git/commit/0002-load-site-font-and-site-tmac-from-etc-groff.patch'>commit</a><a href='/cgit/rpms/groff.git/diff/0002-load-site-font-and-site-tmac-from-etc-groff.patch'>diff</a><a href='/cgit/rpms/groff.git/stats/0002-load-site-font-and-site-tmac-from-etc-groff.patch'>stats</a></td><td class='form'><form class='right' method='get' action='/cgit/rpms/groff.git/log/0002-load-site-font-and-site-tmac-from-etc-groff.patch'>
<select name='qt'>
<option value='grep'>log msg</option>
<option value='author'>author</option>
<option value='committer'>committer</option>
<option value='range'>range</option>
</select>
<input class='txt' type='text' size='10' name='q' value=''/>
<input type='submit' value='search'/>
</form>
</td></tr></table>
<div class='path'>path: <a href='/cgit/rpms/groff.git/tree/'>root</a>/<a href='/cgit/rpms/groff.git/tree/0002-load-site-font-and-site-tmac-from-etc-groff.patch'>0002-load-site-font-and-site-tmac-from-etc-groff.patch</a></div><div class='content'>blob: 163f784ffcccd96db4b6f9b7df610a3789d28fd6 (<a href='/cgit/rpms/groff.git/plain/0002-load-site-font-and-site-tmac-from-etc-groff.patch'>plain</a>)
<table summary='blob content' class='blob'>
<tr><td class='linenumbers'><pre><a id='n1' href='#n1'>1</a>
<a id='n2' href='#n2'>2</a>
<a id='n3' href='#n3'>3</a>
<a id='n4' href='#n4'>4</a>
<a id='n5' href='#n5'>5</a>
<a id='n6' href='#n6'>6</a>
<a id='n7' href='#n7'>7</a>
<a id='n8' href='#n8'>8</a>
<a id='n9' href='#n9'>9</a>
<a id='n10' href='#n10'>10</a>
<a id='n11' href='#n11'>11</a>
<a id='n12' href='#n12'>12</a>
<a id='n13' href='#n13'>13</a>
<a id='n14' href='#n14'>14</a>
<a id='n15' href='#n15'>15</a>
<a id='n16' href='#n16'>16</a>
<a id='n17' href='#n17'>17</a>
<a id='n18' href='#n18'>18</a>
<a id='n19' href='#n19'>19</a>
<a id='n20' href='#n20'>20</a>
<a id='n21' href='#n21'>21</a>
<a id='n22' href='#n22'>22</a>
<a id='n23' href='#n23'>23</a>
<a id='n24' href='#n24'>24</a>
<a id='n25' href='#n25'>25</a>
<a id='n26' href='#n26'>26</a>
<a id='n27' href='#n27'>27</a>
<a id='n28' href='#n28'>28</a>
<a id='n29' href='#n29'>29</a>
<a id='n30' href='#n30'>30</a>
<a id='n31' href='#n31'>31</a>
<a id='n32' href='#n32'>32</a>
<a id='n33' href='#n33'>33</a>
<a id='n34' href='#n34'>34</a>
<a id='n35' href='#n35'>35</a>
<a id='n36' href='#n36'>36</a>
<a id='n37' href='#n37'>37</a>
<a id='n38' href='#n38'>38</a>
<a id='n39' href='#n39'>39</a>
<a id='n40' href='#n40'>40</a>
<a id='n41' href='#n41'>41</a>
<a id='n42' href='#n42'>42</a>
<a id='n43' href='#n43'>43</a>
</pre></td>
<td class='lines'><pre><code>From c6d8bb3e0ebc03274564d7b2c768e9932cc5f79d Mon Sep 17 00:00:00 2001
From: Jan Vcelak &lt;jvcelak&#64;redhat.com&gt;
Date: Tue, 1 Jan 2013 15:33:45 +0100
Subject: [PATCH] load site-font and site-tmac from /etc/groff

Move site-font and site-tmac configuration from /usr/share/groff to
/etc/groff. That allows permanent custom changes. (Symlinking brought
a lot of problems with RPM. This is safer and cleaner.)

Signed-off-by: Jan Vcelak &lt;jvcelak&#64;redhat.com&gt;
<span class="hl kwb">---</span>
 Makefile.in | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/Makefile.in b/Makefile.in
index 7534e56..d97d1b9 100644
<span class="hl kwb">--- a/Makefile.in</span>
<span class="hl kwa">+++ b/Makefile.in</span>
&#64;&#64; -167,7 +167,7 &#64;&#64; fontdir=$(datasubdir)/font
 oldfontdir=$(datasubdir)/oldfont
 
 # `localfontdir' says where local fonts will be installed (as dev*/*).
<span class="hl kwb">-localfontdir=$(dataprogramdir)/site-font</span>
<span class="hl kwa">+localfontdir=&#64;sysconfdir&#64;/groff/site-font</span>
 
 # `legacyfontdir' is for compatibility with non-GNU troff.
 legacyfontdir=/usr/lib/font
&#64;&#64; -179,10 +179,10 &#64;&#64; fontpath=$(localfontdir)$(RT_SEP)$(fontdir)$(RT_SEP)$(legacyfontdir)
 tmacdir=$(datasubdir)/tmac
 
 # `systemtmacdir' says where to install platform-dependent macros.
<span class="hl kwb">-systemtmacdir=$(libprogramdir)/site-tmac</span>
<span class="hl kwa">+systemtmacdir=&#64;sysconfdir&#64;/groff/site-tmac</span>
 
 # `localtmacdir' says where local files will be installed.
<span class="hl kwb">-localtmacdir=$(dataprogramdir)/site-tmac</span>
<span class="hl kwa">+localtmacdir=&#64;sysconfdir&#64;/groff/site-tmac</span>
 
 # `appresdir' says where to install the application resource file for
 # gxditview.
<span class="hl kwb">-- </span>
1.8.0.2

</code></pre></td></tr></table>
</div> <!-- class=content -->
<div class='footer'>generated by <a href='https://git.zx2c4.com/cgit/about/'>cgit v1.1</a> at 2018-03-03 16:33:44 +0000</div>
</div> <!-- id=cgit -->
</body>
</html>
